echoregions/formats/regions2d.py

```diff
@@ -90,6 +91,16 @@ def offset(self, val):
         """Set the depth offset to apply to y values"""
         self._offset = float(val)
 
+    @property
+    def plotter(self):
+        if self._plotter is None:
+            if not self.data:
+                raise ValueError(
+                    "Input file has not been parsed; call `parse_file` to parse."
+                )
```

Comment:
If I understand the current structure correctly, the case when `self.data` does not exist is when people only want to use `get_points_from_region` to get a region from a JSON or CSV file, or from previously parsed data. BUT, I don't see how that line `return self.plotter.get_points_from_region(region, file)` under `Region2D::get_points_from_region()` would work, since there is no corresponding method in `Regions2DPlotter`. I think the design and use case of the `Region2D` class should be reviewed.

It seems a good idea to just parse the input EVR file when `Region2D` is initiated. If I understand correctly, all methods in this class requires `self.data` to exist to work.